webui/react/src/services/apiConfig.ts

```diff
@@ -313,6 +316,40 @@ export const assignRolesToGroup: DetApi<
     }),
 };
 
+export const assignRolesToUser: DetApi<
+  Service.AssignRolesToUserParams,
+  Api.V1AssignRolesResponse,
+  Api.V1AssignRolesResponse
+> = {
+  name: 'assignRolesToGroup',
+  postProcess: (response) => response,
+  request: (params) =>
+    detApi.RBAC.assignRoles({
+      userRoleAssignments: params.roleIds.map((roleId) => ({
+        roleAssignment: { role: { roleId } },
+        userId: params.userId,
+      })),
+    }),
+};
+
+export const removeRoleFromUser: DetApi<
```

Comment:
@gt2345 As part of https://determinedai.atlassian.net/browse/DET-8219 I also added created assign and remove role functions but did not scope them to users or groups. For example, here I just added a `removeRole` that will accept `groupAssignments` or `userAssignments` do you have any thoughts on keeping a generic function or having separate `removeRoleFromUser` and `removeRoleFromGroup` functions? 

I actually do like the separate functionality now that I am seeing it so I will plan to  instead implement a separate  `removeRoleFromGroup` function.

Reply:
Cool, thank you for letting me know